<!DOCTYPE html>
<html class="writer-html5" lang="en" >
<head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
      <link rel="shortcut icon" href="../img/favicon.ico" />
    <title>Lab Introduction - Parisod Lab Dry Guide</title>
    <link rel="stylesheet" href="../css/theme.css" />
    <link rel="stylesheet" href="../css/theme_extra.css" />
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.8.0/styles/github.min.css" />
    
      <script>
        // Current page data
        var mkdocs_page_name = "Lab Introduction";
        var mkdocs_page_input_path = "intro.md";
        var mkdocs_page_url = null;
      </script>
    
    <!--[if lt IE 9]>
      <script src="../js/html5shiv.min.js"></script>
    <![endif]-->
      <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.8.0/highlight.min.js"></script>
      <script>hljs.highlightAll();</script> 
</head>

<body class="wy-body-for-nav" role="document">

  <div class="wy-grid-for-nav">
    <nav data-toggle="wy-nav-shift" class="wy-nav-side stickynav">
    <div class="wy-side-scroll">
      <div class="wy-side-nav-search">
          <a href=".." class="icon icon-home"> Parisod Lab Dry Guide
        </a><div role="search">
  <form id ="rtd-search-form" class="wy-form" action="../search.html" method="get">
      <input type="text" name="q" placeholder="Search docs" aria-label="Search docs" title="Type search term here" />
  </form>
</div>
      </div>

      <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="Navigation menu">
              <ul>
                <li class="toctree-l1"><a class="reference internal" href="..">Home</a>
                </li>
              </ul>
              <p class="caption"><span class="caption-text">Knowledge Base</span></p>
              <ul class="current">
                  <li class="toctree-l1 current"><a class="reference internal current" href="#">Lab Introduction</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../best_practices/">Best Practices</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../r/">R</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../bash/">Command Line</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../vscode/">VScode</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../github/">Source Control with Git and VScode</a>
                  </li>
              </ul>
              <p class="caption"><span class="caption-text">Data Management</span></p>
              <ul>
                  <li class="toctree-l1"><a class="" href="../data-organization.md">Data Organization</a>
                  </li>
                  <li class="toctree-l1"><a class="" href="../data-backup.md">Data Backup</a>
                  </li>
                  <li class="toctree-l1"><a class="" href="../data-sharing.md">Data Sharing</a>
                  </li>
              </ul>
              <p class="caption"><span class="caption-text">IBU</span></p>
              <ul>
                  <li class="toctree-l1"><a class="reference internal" href="../ibu-intro/">Introduction</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../VAST/">Data on IBU</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../ibu-nextflow/">Nextflow</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../ibu-conda/">Conda</a>
                  </li>
              </ul>
              <p class="caption"><span class="caption-text">snakeGATK sequencing workflow</span></p>
              <ul>
                  <li class="toctree-l1"><a class="reference internal" href="../pipeline-overview/">Overview</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../pipeline-trimming/">Trim pipeline</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../pipeline-alignment/">Alignment pipeline</a>
                  </li>
                  <li class="toctree-l1"><a class="" href="../pipeline-GATK.md">Genotyping pipeline</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../pipeline-concordance/">Concordance pipeline</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../pipeline-postGATK/">Post-GATK pipeline</a>
                  </li>
                  <li class="toctree-l1"><a class="reference internal" href="../pipeline-impute/">Imputation pipeline</a>
                  </li>
                  <li class="toctree-l1"><a class="" href="../pipeline-annotation.md">Annotation pipeline</a>
                  </li>
              </ul>
              <p class="caption"><span class="caption-text">Other pipelines</span></p>
              <ul>
                  <li class="toctree-l1"><a class="" href="../pipeline-freebayes.md">Freebayes calling</a>
                  </li>
              </ul>
              <p class="caption"><span class="caption-text">snakeRNAseq workflow</span></p>
              <ul>
                  <li class="toctree-l1"><a class="" href="../rnaseq-overview.md">Overview</a>
                  </li>
                  <li class="toctree-l1"><a class="" href="../rnaseq-trimming.md">Trim pipeline</a>
                  </li>
                  <li class="toctree-l1"><a class="" href="../rnaseq-alignment.md">Alignment pipeline</a>
                  </li>
                  <li class="toctree-l1"><a class="" href="../rnaseq-counting.md">Counting pipeline</a>
                  </li>
                  <li class="toctree-l1"><a class="" href="../rnaseq-diffexp.md">Differential expression pipeline</a>
                  </li>
                  <li class="toctree-l1"><a class="" href="../rnaseq-visualization.md">Visualization pipeline</a>
                  </li>
              </ul>
              <p class="caption"><span class="caption-text">Other things</span></p>
              <ul>
                  <li class="toctree-l1"><a class="" href="../labsite_parisod.md">Parisod Labsite</a>
                  </li>
              </ul>
      </div>
    </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">
      <nav class="wy-nav-top" role="navigation" aria-label="Mobile navigation menu">
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="..">Parisod Lab Dry Guide</a>
        
      </nav>
      <div class="wy-nav-content">
        <div class="rst-content"><div role="navigation" aria-label="breadcrumbs navigation">
  <ul class="wy-breadcrumbs">
    <li><a href=".." class="icon icon-home" aria-label="Docs"></a></li>
          <li class="breadcrumb-item">Knowledge Base</li>
      <li class="breadcrumb-item active">Lab Introduction</li>
    <li class="wy-breadcrumbs-aside">
          <a href="https://github.com/parisodlab/dry-guide/edit/master/docs/intro.md" class="icon icon-github"> Edit on GitHub</a>
    </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
            <div class="section" itemprop="articleBody">
              
                <p>Information for all members of the Parisod lab</p>
<p>Department of Biology, University of Fribourg
PER23-Ecology &amp; Evolution building, Chemin du Musée 15
PI : Prof. Christian Parisod, PER23-102, 0 26 300 88 52, <a href="mailto:christian.parisod@unifr.ch">christian.parisod@unifr.ch</a>
Administrative assistant: Eirini Maikanti, PER23-001, 026 300 88 50, <a href="mailto:eirini.maikanti@unifr.ch">eirini.maikanti@unifr.ch</a>
Tech assistants: Dr. Boris Egger (PER05-0.340G, <a href="mailto:boris.egger@unifr.ch">boris.egger@unifr.ch</a>)
Alain Werro (PER04-01.117, 026 300 88 12)
Gardner: Alain Müller(<a href="mailto:alain.mueller@unifr.ch">alain.mueller@unifr.ch</a>), Benoît Clément (Alpine Plants, <a href="mailto:benoit.clement@unifr.ch">benoit.clement@unifr.ch</a>)</p>
<p>All HR-related matter as well as room authorizations must be coordinated with the administrative assistant. An access to <strong>my.unifr.ch</strong> through your campus account will simplify many administrative tasks. Also see <a href="https://www.unifr.ch/campus/en/personal-support/administrative.html">https://www.unifr.ch/campus/en/personal-support/administrative.html</a></p>
<p>Our lab deals with molecular, computational and living resources. It involves your contribution by keeping things updated and through lab/greenhouse duties planned in advance. Christian will introduce you to the necessary infrastructures for your project and our elabFTW database for the storage of data and procedures (see §3.6ff). Start your stay by being known in our system and continue it with zealous updates.</p>
<p>We use lab meetings to promote general coordination in the lab. A google-calendar (<a href="&#109;&#97;&#105;&#108;&#116;&#111;&#58;&#108;&#97;&#98;&#112;&#97;&#114;&#105;&#115;&#111;&#100;&#64;&#103;&#109;&#97;&#105;&#108;&#46;&#99;&#111;&#109;">&#108;&#97;&#98;&#112;&#97;&#114;&#105;&#115;&#111;&#100;&#64;&#103;&#109;&#97;&#105;&#108;&#46;&#99;&#111;&#109;</a>) and Slack (Parisod Lab) is used to make a note about your vacation plans ahead of time, as to organize replacement if necessary. If you must unexpectedly take day(s) off (e.g. illness or unanticipated circumstances), notify Christian directly by e-mail. E-mails to all lab members can be sent through <a href="mailto:biol-parisodgp@unifr.ch">biol-parisodgp@unifr.ch</a>.</p>
<p>Two series of scientific seminars are taking place during semesters: Departmental seminars with mostly external guests every Tuesdays (11.15 am, PER04-0.110) and Environmental Biology seminars by internal collaborators every Thursdays (15.15 pm, PER04-0.110). Your attendance and participation are welcome! Register to to such and other teaching-related activities and get information through <a href="https://moodle.unifr.ch/">https://moodle.unifr.ch/</a></p>
<p><strong>Table of Content</strong></p>
<ol>
<li>Lab members: expectations &amp; responsibilities<ol>
<li>Everyone</li>
<li>PI</li>
<li>Postdocs</li>
<li>PhD candidates</li>
<li>MSc students</li>
<li>Lab managers</li>
<li>Internships s.l.</li>
</ol>
</li>
<li>Code of conduct<ol>
<li>Harassment</li>
<li>Reproducible research</li>
<li>Responsible research</li>
</ol>
</li>
<li>Organization of the lab<ol>
<li>Access to infrastructure</li>
<li>Working hours</li>
<li>PI availability</li>
<li>Lab meetings</li>
<li>Lab travels</li>
<li>Lab archives &amp; elabFTW</li>
<li>Lab infrastructure</li>
<li>Lab rules</li>
</ol>
</li>
</ol>
<p><strong>1. Lab member expectations &amp; responsibilities</strong></p>
<p><em>1.1 Everyone (full-time, part-time, and undergraduate internship members)</em></p>
<p>Keep your lab and office space tidy on a daily basis. Take your share for team-level activities.</p>
<p>Do work that you are proud of. Do work that others will care about. Double-check your work (good science builds on solid evidence). Would you feel your work does not meet these standards, let’s discuss possible solutions.</p>
<p>Science is a marathon. Academia has a specific flavor, but it is still a job. Be professional. Do not come into the lab if you are sick. Take personal time and vacation when needed and cultivate a life outside of the lab. There may be times that you have to work longer or harder to finish something, but you should feel it balanced out over time.</p>
<p>We are a team. Be supportive of your lab mates. Attend lab meetings and group activities. Respect each others' strengths, weaknesses, differences... Communicate openly. Work independently when you can, but make the best of working next to peers and ask for help when you need it. Share your knowledge. Typical workflow in the face of a question: (i) consult documentation, (ii) ask google, (iii) ask peers, (iv) ask direct mentor, (v) ask PI. It can be shortcut.</p>
<p>We work within an academic system that is fundamentally self-driven. Participate to it with curiosity. Among the several educational events that target a broad audience, regular series of seminars shall normally be attended. An overview is to be found on moodle (register as participant) and in a weekly newsletter.</p>
<p>If you have an issue with another lab member that cannot be solved through direct communication, please talk with Christian. If you have an issue with Christian, please reach out the Mediation Service of UNIFR: <a href="https://www.unifr.ch/uni/en/administration/mediation-office.html">https://www.unifr.ch/uni/en/administration/mediation-office.html</a></p>
<p>Before quitting the lab for new adventures, sort and archive your documents appropriately, clean your lab and office space, make sure that your samples are properly stored (labelled with your name and a date!) and referenced in our database. Check with your direct supervisor. See § 4 for further details.</p>
<p><em>1.2 PI (<a href="&#109;&#97;&#105;&#108;&#116;&#111;&#58;&#99;&#104;&#114;&#105;&#115;&#116;&#105;&#97;&#110;&#46;&#112;&#97;&#114;&#105;&#115;&#111;&#100;&#64;&#117;&#110;&#105;&#102;&#114;&#46;&#99;&#104;">&#99;&#104;&#114;&#105;&#115;&#116;&#105;&#97;&#110;&#46;&#112;&#97;&#114;&#105;&#115;&#111;&#100;&#64;&#117;&#110;&#105;&#102;&#114;&#46;&#99;&#104;</a>)</em></p>
<p>All of the above, plus you can expect me to:</p>
<p>Maintain a vision of where the lab is going. Apply for and secure the funding necessary to keep the lab going. Sketch out the strategic plan on a yearly basis to keep you on track with your goals.</p>
<p>Be available (also see §3.3) to meet with you as regularly as needed to discuss your research and make it progress towards publication. Work with you to regularly develop a mentoring and research plan tailored to your interests, needs, and goals. We will set the definition of "regularly" together across your project.</p>
<p>Support your career development by introducing you to networks, writing recommendation letters for you, providing you with opportunities to attend conferences and promoting your work in talks and papers. We will have a yearly meeting to focus on professional achievements and perspectives.</p>
<p>I am happy to discuss any concern that may be influencing your work. We can always discuss extra support related to time management and productivity and brainstorm possible solutions.</p>
<p><em>1.3 Postdocs</em></p>
<p>All of the above, plus you will be expected to:</p>
<p>Generally develop your own independent line of research and mentor early researchers on their projects, when asked or when appropriate.</p>
<p>Apply for external funding. Apply for jobs (academic or industry) as soon as you are "ready" and/or at least six months before the term of your contract. We can discuss ways of making sure that you are getting the training you need, while still doing excellent research.</p>
<p>I encourage you to seek out opportunities to present your research to the department, research community, or general public. If you are going to give a presentation (including posters and talks), please request a practice presentation to the lab at least one week ahead of time.</p>
<p><em>1.4 PhD candidates</em></p>
<p>All of the above, plus you will be expected to:</p>
<p>Develop a line of dissertation research. Ideally, your dissertation research will consist of at least three chapters (i.e. independent studies/experiments) that can be packaged into one thesis document.</p>
<p>Develop your skills (scientific, technical, transferable). Work with undergraduate students, as to not only promote data collection but also experience team managing and mentoring. Apply for external funding, as a valuable learning experience.</p>
<p>Question the type of career you want to pursue (e.g. research-focused or teaching-focused, non-academic jobs like data science or science writing). We can brainstorm ways of making sure you are getting the training that you need. Apply for jobs at least six months before the term of your contract.</p>
<p>Prioritize time for research (at the end of the training period, you need to have a completed dissertation). Stay up-to-date on any deadlines that you need to meet to fulfill departmental requirements (e.g. funding applications, master’s defense, teaching, … ).</p>
<p><em>1.5 MSc students</em></p>
<p>All of the above, plus you will be expected to:</p>
<p>Develop a line of dissertation research. Ideally, your dissertation will consist of complementary analyses using data that you have produced and that can ideally be packaged into a scientific publication.</p>
<p>Develop your organization and skills (scientific, technical, transferable).</p>
<p>Question the type of career you want to pursue (e.g. research-focused or teaching-focused, non-academic jobs like data science or science writing).</p>
<p><em>1.6 Lab managers</em></p>
<p>All of the above, plus you will be expected to:</p>
<p>Maintain the list of lab members updated (including the biol-parisodgp mailing list, elabWTF, slack et al, and access to lab infrastructures through magnetic locks by informing Eirini with name and duration, ...)</p>
<p>Maintain the lab internal database and routines updated, with proper archiving of samples and protocols. Order main consumables. Keep this manual up to date.</p>
<p>Help to maintain an atmosphere of professionalism within the lab. Oversee the scheduling and training of newcomers in the lab. Ensures proper reporting before the leaving of a lab member.</p>
<p>Assist other lab members with data collection, storage or analysis. Work on your own research project.</p>
<p><em>1.7 Undergraduate students and research assistants (s.l.)</em></p>
<p>Undergraduate research assistants can earn credits or money for their contribution to our lab routines. Enquiries can be made by writing an e-mail to Christian or approaching a lab member.</p>
<p>If you are hired as a research assistant (i.e. contract, for typically 4-8 weekly hours or specified time blocks), you will be under the daily supervision of a direct mentor with whom you will determine your schedule and all of the above concerns you.</p>
<p><strong>2. Code of conduct</strong></p>
<p>In addition to the general expectations laid out above, I am dedicated to making our lab a safe, inclusive, and welcoming environment for all.</p>
<p><em>2.1 Harassment</em></p>
<p>All members of the lab, along with visitors, are expected to agree with a decent code of conduct. I expect all lab members to treat one another with respect and to be sensitive to how one’s words and actions impact others. We also know that science benefits from diverse perspectives.</p>
<p>Members asked to stop any harassing behavior are expected to comply immediately. “Qui vient trop près va trop loin. Wer zu nah kommt, geht zu weit». If you are being harassed, notice that someone else is being harassed, or have any other concerns, please contact Christian or the UNIFR mediation department.</p>
<p><em>2.2 Reproducible research</em></p>
<p>Reproducible research is research that can be exactly reproduced (i.e. replicability as the ability to generate the same results again). It is expected that all of our research can be justified, at minimum, as reproducible (incl. own testing when possible). That requires to be organized and possess sufficient foresight to document each step of your research process. See §3.6 for our internal database.</p>
<p>Scientific integrity <a href="https://www.unifr.ch/scimed/en/integrity">https://www.unifr.ch/scimed/en/integrity</a></p>
<p><em>2.3 Responsible research</em></p>
<p>Authorship credit should reflect the individual's contribution to the study. An author is considered anyone contributing with initial research design, data collection and analysis, manuscript drafting, and final approval. All authors assume responsibility for the publication, making sure that the data they contributed are accurate. The submitting author makes sure that all deserving authors have been credited, that all authors have given their approval; and handles responses to inquiries during and after publication.</p>
<p>Authorship will usually be discussed prior to the beginning of a new project to clearly set expectations. Changes to authorship may occur over the course of a project if new collaborators become involved or if someone is not fulfilling their planned role. In general, I expect that PhD candidates and postdocs in the lab will be first authors on publications on which they are the primary lead, and I will be the last author.</p>
<p>As for publications out of the lab, we generally try to avoid thin slices and tend to privilege integrative evidence supporting a complete story as much as possible. Would you feel that our targeted high-hanging fruits can hardly get harvested, let’s discuss how to handle the situation to possibly avoid starvation.</p>
<p>For projects that have required significant lab resources, "ownership" of data to the producer can be ended on mutual agreement or maximum 2 years after collection. At that point, the PI reserves the right to re-assign the project (or not) as needed to expedite publications. This is to avoid situations of long languishing datasets, while supporting priority.</p>
<p><strong>3. Organization of the lab</strong></p>
<p><em>3.1 Access to infrastructure</em></p>
<p>Lab safety <a href="https://www.unifr.ch/scimed/en/safety/commissions">https://www.unifr.ch/scimed/en/safety/commissions</a></p>
<p>Authorization for staff to enable entry through the main entrance of the Ecology &amp; Evolution building (PER23; Chemin du Musée 15) is requested through the administrative assistant. Same for physical keys. When you leave the lab, make sure to return your key (and get any deposit back).</p>
<p><em>3.2 Working hours</em></p>
<p>One benefit of a career in academic research is typical flexibility. In your contract, 100% legally means being employed for 42 hours a week. You are typically not required to work over-time. I expect you to be in the lab, at minimum, multiple weekdays at peak hours (10am and 2pm or so) to promote interactions.</p>
<p><em>3.3 PI availability</em></p>
<p>In addition to poking my head into the lab or your office regularly, I will be working on campus and available for meetings most days of the week. My door is semi-open when I can be available. Feel free to pop in so that we can set ad-hoc meetings to shortly discuss.</p>
<p>As I may turn my “notifications” off when concentrating on something else, you can always send me an e-mail. I do my best to shortly answer and will not mind receiving a reminder after more than 48h. If you need something by a particular deadline, please indicate it. Early information (at least, one week beforehand) should enable me to allocate sufficient time and meet your deadline.</p>
<p>Recommendation letters: I will usually write a letter for any lab member having spent multiple months in the lab. Your request (ideally, two weeks ahead of time) should include the deadline and your current CV as well as any relevant instruction. To ensure that I do not miss any important details, I may ask you for a draft, which I would then revise consistently.</p>
<p><em>3.4 Lab meetings</em></p>
<p>Lab meetings, every Tuesday (from 10.15 am, PER23) last 45minutes. If more time is necessary, we will schedule an extraordinary meeting. Every second slot will be focused on project presentations going over new data or methods. Every other slot will be a journal club with the Flatt group. All full-time lab members are expected to attend lab meetings. Part-time internships are welcome.</p>
<p>To ensure effective lab meetings, it is recommended to adequately prepare, especially when presenting. This promotes not only your progress, but also fosters useful feedback. This is also a privileged time to set bilateral meetings and foster cooperation.</p>
<p><em>3.5 Lab travels</em></p>
<p>The lab will typically pay for full-time lab members to work in the lab of collaborators or to present their work at conferences (e.g. National, such as Biology’2i; International). In general, presented work should be original and appropriate for the targeted conference. Wishes must be discussed beforehand as they may depend on the availability of funds. Transparent reimbursement of your work-related costs must in all cases be justified by invoices and must be coordinated with administrative assistant based on founding source). Lab members can apply for other sources of funding available to them (e.g. Swiss Botanical Society).</p>
<p><em>3.6 Lab archives &amp; elabFTW</em></p>
<p>We have three alternatives for data storage:</p>
<p>-Common server of Biology, for light data</p>
<p>MacOS/linux: smb://common.unifr.ch/biol/_Ecologie/Parisod_group/</p>
<p>Windows: [\\common.unifr.ch\biol\_Ecologie\Parisod_group\](file:///\common.unifr.ch\biol_Ecologie\Parisod_group)</p>
<p>You can for instance find environmental conditions of the greenhouse (Greenhouse folder: temperature, light, humidity), lab meeting reports (Lab_meetings folder), posters and talks (Presentations folder), research documents (Projects, Protocols, Publications, Scripts) in Research_doc folder, and teaching documents. Please, think about feeding it with your contributions.</p>
<p>-Big data server, for heavy data matching the moto “write once read many”</p>
<p>MacOS/linux: smb://bigdata.unifr.ch/science/biol/groupe_parisod/</p>
<p>Windows: [\\bigdata.unifr.ch\science\biol\groupe_parisod\](file:///\bigdata.unifr.ch\science\biol\groupe_parisod)</p>
<p>You can find the main folders Archives (to put all data that are currently in a server such as NCBI or EMBL-EBI or data that are not used anymore), Raw_data (to put data that are still being used), Shared (contains protocols, results and software automatically stored from ElabFTW), Users (space to store your own data).</p>
<p>-Electronic lab notebook, to document experiments and data stored in the bigdata server</p>
<p><a href="https://parisodlab.unifr.ch:8888">https://parisodlab.unifr.ch:8888</a> use login and password of the UNIFR (LDAP) once registered (ask Patrick Favre for subscription)</p>
<p>ElabFTW allows to record your experiments and can be shared (or not) with others. It allows important lab information, such as chemicals, protocols and scripts, to be stored in a database. All contents can be linked to one another (e.g. experiment to databases, experiment to experiments).</p>
<p>Parisod:Pop: Samples from natural population (collected in the wild). Each sample included in our collection should be recorded in Parisod Natural with the following fields: collector’s name, date, locality, coordinates, possibly some comments on subsequent fate (e.g. RNA/DNA extraction).</p>
<p>Parisod:Experimental: Samples from experimental populations (crossings...). Each cross samples should be recorded in Parisod experimental with fields (collactor name, date, crossing…).</p>
<p><em>3.7 Lab rules</em></p>
<p>A detailed protocol is to be followed across labs at PER23. It must be read and signed beforehand. Note that the lab 305 is PCR free.</p>
<p>Programming of conditions in the LED growth chamber (PER23-104) and growth cabinets (PER23-205a): Patrick Favre.</p>
<p>Greenhouses: Keys available PER23-008. Programming of conditions: Alain Werro.</p>
              
            </div>
          </div><footer>
    <div class="rst-footer-buttons" role="navigation" aria-label="Footer Navigation">
        <a href=".." class="btn btn-neutral float-left" title="Home"><span class="icon icon-circle-arrow-left"></span> Previous</a>
        <a href="../best_practices/" class="btn btn-neutral float-right" title="Best Practices">Next <span class="icon icon-circle-arrow-right"></span></a>
    </div>

  <hr/>

  <div role="contentinfo">
    <!-- Copyright etc -->
  </div>

  Built with <a href="https://www.mkdocs.org/">MkDocs</a> using a <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>.
</footer>
          
        </div>
      </div>

    </section>

  </div>

  <div class="rst-versions" role="note" aria-label="Versions">
  <span class="rst-current-version" data-toggle="rst-current-version">
    
        <span>
          <a href="https://github.com/parisodlab/dry-guide" class="fa fa-github" style="color: #fcfcfc"> GitHub</a>
        </span>
    
    
      <span><a href=".." style="color: #fcfcfc">&laquo; Previous</a></span>
    
    
      <span><a href="../best_practices/" style="color: #fcfcfc">Next &raquo;</a></span>
    
  </span>
</div>
    <script src="../js/jquery-3.6.0.min.js"></script>
    <script>var base_url = "..";</script>
    <script src="../js/theme_extra.js"></script>
    <script src="../js/theme.js"></script>
      <script src="../search/main.js"></script>
    <script>
        jQuery(function () {
            SphinxRtdTheme.Navigation.enable(true);
        });
    </script>

</body>
</html>
